src/theme/foundations/breakpoints.ts

```diff
@@ -16,7 +21,11 @@
 export default {
   sm: "20em",
   md: "37.5em",
+  // Temporary and just for the header
+  mh: headerBreakpoints.mh,
   lg: "60em",
+  // Temporary and just for the header
+  lh: headerBreakpoints.lh,
```

Comment:
I just noticed these.

Reply:
Yeah, not ideal but for now it's ok. No other components will use these breakpoints.